### PR TITLE
[GITHUB-1] Initial NTP role import

### PR DESCRIPTION
**Pull request by wojtek-oledzki**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# NTP
```
Comment (lobsterdore) [inline README.md:1]: Do we want the expanded style readme that we have in the other roles here?

**Comment by lobsterdore** [comment]
Just one small comment, other than that this looks good.

File: README.md (hunk 1 above)
Comment (wojtek-oledzki) [inline README.md:1]: I've missed that one - adding now...

**Comment by wojtek-oledzki** [comment]
@lobsterdore proper README added :)

**Comment by lobsterdore** [comment]
LGTM :+1:

**Pull request merged**